src/status_im/subs/wallet/wallet.cljs

```diff
@@ -261,22 +263,22 @@
  :wallet/settings-keypairs-accounts
  :<- [:wallet/keypairs]
  (fn [keypairs [_ format-options]]
-   (let [grouped-keypairs     (group-by :lowest-operability keypairs)
+   (let [grouped-keypairs     (->> keypairs
+                                   vals
+                                   (group-by :lowest-operability))
          operable-keypair-ids (->> (concat (:fully grouped-keypairs)
                                            (:partially grouped-keypairs))
-                                   (map :key-uid)
-                                   (into #{}))
-         missing-keypair-ids  (->> (map :key-uid (:no grouped-keypairs))
-                                   (into #{}))]
-     {:operable (->> keypairs
-                     (filter #(contains? operable-keypair-ids (:key-uid %)))
+                                   (mapv :key-uid))
+         missing-keypair-ids  (mapv :key-uid (:no grouped-keypairs))]
```

Comment:
Unless we strictly need a vector, I think is better to just use `map` and keep the operations lazy